bin/i18n/sync-codeorg-in

```diff
@@ -7,57 +7,106 @@
 
 require File.expand_path('../../../pegasus/src/env', __FILE__)
 require 'fileutils'
+require 'json'
 require 'yaml'
 require 'tempfile'
 
+require_relative 'utils'
+
 def pull_in_files_for_translation
   localize_instructions
-  exec "bin/i18n-codeorg/in.sh"
+  run_bash_script "bin/i18n-codeorg/in.sh"
 end
 
-def find_instruction(f)
-  pattern = /^\s*"instructions": (".*?"),?\n$/
-  f.each_line do |line|
-    matches = line.match pattern
-    if matches
-      return matches.captures.first
-    end
-  end
-  nil
-end
-
-def copy_instructions_to_yml(instructions)
-  File.open("dashboard/config/locales/instructions.en.yml", "w+") do |f|
-    f.write(({ "en" => { "data" => { "instructions" => instructions }}}).to_yaml)
+def copy_to_yml(label, data)
+  File.open("dashboard/config/locales/#{label}.en.yml", "w+") do |f|
+    f.write(({ "en" => { "data" => { label => data }}}).to_yaml)
     File.write(f, File.read(f))
   end
 end
 
 def reformat_quotes
-  temp_file = Tempfile.new("tempinstructions.yml")
-  File.open("dashboard/config/locales/instructions.en.yml", "r") do |f|
-    f.each_line { |line| temp_file.puts line.gsub("'\"", '"').gsub("\"'", '"').gsub("''", "'")}
+  filenames = ["instructions", "markdown_instructions"]
```

Comment:
I didn't think about this before, but if the markdown strings are in a .yml Crowdin might not expect it to be markdown and may not process it the same. For example, they usually highlight code as green as an indicator to translators that it shouldn't be translated. I'm not sure if it'll be able to recognize the same here, but we'll test it out!